src/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -139,6 +139,7 @@ def wait_for_gridscan_valid(fgs_motors: FastGridScan, timeout=0.5):
             f"Scan invalid: {scan_invalid} and position counter: {pos_counter}"
         )
         if not scan_invalid and pos_counter == 0:
+            hyperion.log.LOGGER.info(f"Scan invalid check: SUCCESSFUL")
```

Comment:
Should: I agree this needn't be an f-string. I also think the all upper case is a bit OTT:
```suggestion
            hyperion.log.LOGGER.info("Gridscan scan valid and position counter reset")
```